sql/core/src/main/scala/org/apache/spark/sql/internal/SessionState.scala

```diff
@@ -166,8 +166,8 @@ private[sql] class SessionState(sparkSession: SparkSession) {
 
   def executePlan(plan: LogicalPlan): QueryExecution = new QueryExecution(sparkSession, plan)
 
-  def invalidateTable(tableName: String): Unit = {
-    catalog.invalidateTable(sqlParser.parseTableIdentifier(tableName))
+  def refreshTable(tableName: String): Unit = {
```

Comment:
To be honest, I still think `invalidateTable` is a right name. We are not doing `refresh`